unix/Makefile

```diff
@@ -74,10 +77,14 @@ ifeq ($(MICROPY_FORCE_32BIT),1)
 # starting with linux-libc-dev:i386
 ifeq ($(MICROPY_PY_FFI),1)
 ifeq ($(UNAME_S),Linux)
+ifeq ($(UNAME_M), armv7l)
+CFLAGS_MOD += -I/usr/include/arm-linux-gnueabihf
```

Comment:
Did you try your patch on a 64-bit ARM machine?

MICROPY_FORCE_32BIT=1 should only be needed for 64-bit machines (and should be harmless on 32-bit machines).

This block of code here in the Makefile is specific to the case where you're on a 64-bit machine, you used MICROPY_FORCE_32BIT=1, and you also want modffi.  In the case of a 32-bit RPi, it should work without your addition.  In the case of a 64-bit RPi, I don't know what happens, but I would have thought that, if anything, you'd want to have `if ($(UNAME_M),armv8)`.